client/dashboard/sites/index.tsx

```diff
@@ -139,6 +140,13 @@ const DEFAULT_FIELDS: Field< Site >[] = [
 		},
 		enableSorting: false,
 	},
+	{
+		id: 'last-published',
+		label: __( 'Last published' ),
+		render: ( { item }: { item: Site } ) =>
+			item.options?.updated_at ? <TimeSince date={ item.options.updated_at } /> : '',
+		enableSorting: true,
```

Comment:
Seems like we need to provide a `getValue` field that is supposed to be sorted.

Reply:
Right you are, cb2b40c02f97a85977be3a31ce9fd2fa1ac6461a 👍